### ISSUE TITLE
A way to detect if a link opens in new window?

### ISSUE THREAD

**Issue by panda0603** [react-native-community/react-native-webview#1246]
At the moment, a link with new window opens within same page in react native webview. 

We need a way to detect if the link opens in new window or not, in onShouldStartLoadWithRequest prop so that we can interrupt and handle links that opens in new tab. 

Is there any workaround so far? 

I have checked the 'target' in request object in onShouldStartLoadWithRequest, but it just returns some number not related to new window.

**Comment by binhndicts** [comment]
@panda0603 Hi, I think it's a limitation of the module.
Please try to see my comment here https://github.com/react-native-community/react-native-webview/issues/1111#issuecomment-599279960
Hope that it can gives you some ideas to solve your problem

**Comment by panda0603** [comment]
@binhndicts Hey thanks for your solution.
I'd really like native solution but since I do not know much about native side, I had to implement through javascript injection.
What I did was look for a tags with target properties, and send post message with link url(at the same time stop navigate)when user clicks on the link. Then I would hanlde that post message on react native. Maybe half solution since there are other ways than a tags with target properties that open with new windows, but works most of the time.
